### ISSUE TITLE
Create Wordpress Gutenberg Block for All Shortcodes

### ISSUE THREAD

**Issue by weismannweb** [mautic/mautic-wordpress#119]
Any interest in this?

**Comment by weismannweb** [comment]
https://www.gcsdesign.com/converting-a-shortcode-to-a-gutenberg-block/#:~:text=Using%20the%20shortcode%20function,is%20used%20for%20the%20shortcode.

**Comment by shulard** [comment]
Hello !

Aren't the shortcodes still supported ? Introducing Gutenberg blocks will imply adding some JS things and this plugin will be more complex to maintain… I'm not sure it's relevant to make that conversion now. Maybe you can ask on Mautic support forum if it's an interesting thing to be done for our current users.

**Comment by weismannweb** [comment]
yes they are still supported but blocks is the path forward and i imagine at some point it will be needed, also shortcodes are sometimes not supported in other shortcodes or some blocks i think unless you use a hook and do_shortcode on that content via a filter, blocks are much more user friendly too, but your right this would be harder to maintain, maybe you can make me the appropriate hooks (unless wp hooks can do it) so that i can turn your code into blocks with a seperate plugin?  i am also interested in doing the counter from #114
